Formula/ibex.rb

```diff
@@ -12,37 +22,59 @@ class Ibex < Formula
     sha256 "41327b6a0da9b8b2ed888c3c2c33d5f6ff061ea13eceb4b9e8b6e660049d624d" => :mountain_lion
   end
 
-  option "with-java", "Build Java bindings for Choco solver."
+  option "with-java", "Enable Java bindings for CHOCO solver."
+  option "with-ampl", "Use AMPL file loader plugin"
```

Comment:
Yes, `--with-java` and `--with-ampl` are mutually exclusive options. The other plugins are built by default (so that a `brew install ibex` will be sufficient for most users).